pkg/cmd/admin/node/listpods.go

```diff
@@ -0,0 +1,101 @@
+package node
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+)
+
+type ListPodsOptions struct {
+	Options *NodeOptions
+}
+
+func (l *ListPodsOptions) AddFlags(cmd *cobra.Command) {
+	kcmdutil.AddPrinterFlags(cmd)
```

Comment:
Seems like this is worth an upstream pull to make it take a `FlagSet`.  It's minor, but that's the object it's actually operating against.